e2e/tests/steps_init_test.go

```diff
@@ -45,189 +35,164 @@ const (
 	chainId    = "0001"
 )
 
-func init() {
-	cs, err := getClientset()
-	if err != nil {
-		logger.Fatal().Err(err).Msg("failed to get clientset")
-	}
-	clientset = cs
-	vkmap, err := pocketk8s.FetchValidatorPrivateKeys(clientset)
+type rootSuite struct {
+	gocuke.TestingT
+
+	// validatorKeys is hydrated by the clientset with credentials for all validators.
+	// validatorKeys maps validator IDs to their private key as a hex string.
+	validatorKeys map[string]string
+	// clientset is the kubernetes API we acquire from the user's $HOME/.kube/config
+	clientset *kubernetes.Clientset
+	// validator holds command results between runs and reports errors to the test suite
+	validator *validatorPod
+	// validatorA maps to suffix ID 001 of the kube pod that we use as our control agent
```

Comment:
Remove this comment?

Reply:
Nice catch - thanks! :raised_hands: I'll track this change locally and include it in a forthcoming PR.